Documentation/project-docs/linux-performance-tracing.md

```diff
@@ -89,7 +102,7 @@ Traces are best viewed using PerfView on Windows.  Note that we're currently loo
 2. Download PerfView from <http://aka.ms/perfview>.
 3. Run PerfView.exe
 
-	> ```bash
+	> ```
```

Comment:
nit: It would be nice to keep ```` ```bash ```` or better or better ```` ```sh ```` for syntax highlighting (generally a good idea to keep code blocks properly annotated on GH)